python/src/wirecraft_server/server.py

```diff
@@ -48,30 +42,35 @@ def _wait_next_refresh(self) -> bool:
             # TODO: implement server slow down (increase delay in case of poor performances)
             logger.warning("Can't keep up ! %ss behind the normal refresh", -round(_wait, 3))
             self._last_refresh = _now
-            return self._stop.is_set()
+            # return self._stop.is_set()
```

Comment:
Do you plan to implement a way to close the server from the godot client?

Reply:
Not really. If we want the server to be stopped from the client, we would also want a way to start the server from the client.

If we use python, we will likely need a python interpreter & libraries to be part of our executable.
If we use docker, we will need the user to have docker installed in the system.

Maybe we can use something like nuitka, or at least pyinstaller, to make python executable and embed this in our godot app, but it looks like too much work for only a little value.

So, I think we will just make the server and client independant.
It is pretty unusual to have a level-based game where we needs a running server to play, but... 🤷